src/__test__/ReducerTests/authreducerTest.test.js

```diff
@@ -1,52 +1,177 @@
 import { actionTypes } from '../../constants/actionTypes';
-import AuthReducer from '../../reducers/authreducer'
-import { expect } from 'chai'
+import AuthReducer from '../../reducers/authreducer';
+import { expect } from 'chai';
 
 const initialState = {
-  userregistered: false,
-  token_: null,
-  message: null,
-  userlogin:false,
-}
+  user: {
+    token: '',
+    message:'',
+    authenticated:false,
+    error: ''
+  },
+  request: {
+    loading:true,
+    error: '',
+  }
+};
 
 describe ('Authentication Reducers', () => {
-    it('CASE: REGISTER_USER_FULFILLED', () => {
-
-      const action = {
-        type: "REGISTER_USER",
-        payload: {
-            data: {
-              token: 'awesome',
-              message: 'Successfully Registered'
-            }
+  it('CASE: REGISTER_USER_FULFILLED', () => {
```

Comment:
why are you hardcoding the actionType string and yet you have imported actionTypes?  i think using the imported ones can help eradicate simple mistake. you could instead use `actionTypes.RequiredActionType`. e.g 
         `actionTypes.REGISTER_USER`